src/rtpdepacketizer.cpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2024 Paul-Louis Ageneau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#if RTC_ENABLE_MEDIA
+
+#include "rtpdepacketizer.hpp"
+#include "rtp.hpp"
+
+#include "impl/logcounter.hpp"
+
+#include <cmath>
+#include <cstring>
+
+namespace rtc {
+
+void OpusRtpDepacketizer::incoming([[maybe_unused]] message_vector &messages,
+                                   [[maybe_unused]] const message_callback &send) {
+
+	messages.erase(std::remove_if(messages.begin(), messages.end(),
+	                              [](message_ptr message) {
+		                              // Filter RTCP
+		                              if (message->type == Message::Control) {
+			                              return true;
+		                              }
+
+		                              if (message->size() < sizeof(RtpHeader)) {
+			                              PLOG_VERBOSE << "RTP packet is too small, size="
+			                                           << message->size();
+			                              return true;
+		                              }
+
+		                              return false;
+	                              }),
+	               messages.end());
+
+	for (auto &message : messages) {
+		auto pkt = reinterpret_cast<const rtc::RtpHeader *>(message->data());
+		auto headerSize = sizeof(rtc::RtpHeader) + pkt->csrcCount() + pkt->getExtensionHeaderSize();
+		message->erase(message->begin(), message->begin() + headerSize);
```

Comment:
I think it's safer to create a new message here. Also, it's consistent with how packetizers operate.
```suggestion
		message = make_message(message->begin() + headerSize, message->end());
```